### DISCUSSION TITLE
Staging link

### DISCUSSION THREAD

**Discussion by TereCodes** [t1-loanwise/loan-wise-Group1-Team1A#6] [General]
https://loan-wise-dev.netlify.app/

@MUFTIATBAKARE 
@moni-nancy 
@Temidayo24 
@Teewhybells 
@EkeUkeni 
@MiriamOkpalaeke

**Comment by tundeojediran** [comment]
Thanks @TereCodes 
Please tag your team mates also

**Reply by tundeojediran** [reply]
@Ogfejiro please post your team's staging link

**Reply by TereCodes** [reply]
> Thanks @TereCodes Please tag your team mates also

How do I do that please?

**Reply by tundeojediran** [reply]
@TereCodes use the @ symbol and select your team name

**Comment by Ogfejiro** [comment]
How do I past staging link please

**Reply by tundeojediran** [reply]
@Ogfejiro, you need to create a new discussion. Or ask Grace how she did hers.
